infra.bs

```diff
@@ -239,6 +239,13 @@ paths would be an insurmountable task and not productive as they would not stand
 as well as the single algorithm would. Therefore performance is best left as a field to compete
 over.
 
+<p tracking-vector>A document using the Infra Standard should not enforce specific limits on
+unconstrained inputs to allow for competition among user agents and foster ever greater data
```

Comment:
The flip side of competition among user agents is that web pages might exceed one browser's limits while only testing on a larger browser. We should probably check with a Firefox person that they're ok with that problem in this case?

Reply:
Or pages rely on a particular limit. The note should probably talk about this sometimes leads to a particular limit being defined.

Another thing I realized is that by not addressing limits we don't define the error handling for hitting them either. At some point there was some ambition to address that for out-of-memory in JS, but not enough to actually make it happen.